examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/CHIPToolActivity.kt

```diff
@@ -28,7 +28,7 @@ import android.widget.Toast
 import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
-import chip.devicecontroller.NetworkCredentials
+import chip.devicecontroller.NetworkCredentialsParcelable
```

Comment:
Woops missed this at first; honestly I'dj ust put this here with CHIPToolActivity, in the same package, rather than having it in devicecontroller

There's LOTS of things that we use in devicecontroller that we end up serializing & just have our own implementations, that's a common pattern, I think it's reasonable to have the implementer have their own thing vs having a shared parcellable

Reply:
yep, just remove device controller here. thanks